docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -21,10 +21,21 @@ const { buildNavBar } = require("./build-api-nav");
 const { renderAlertNode, renderBlockQuoteNode, renderTableNode } = require("./custom-renderers");
 const { createHugoFrontMatter } = require("./front-matter");
 
-const apiReportsDirectoryPath = path.resolve(__dirname, "..", "_api-extractor-temp", "_build");
-const apiDocsDirectoryPath = path.resolve(__dirname, "..", "content", "docs", "apis");
+/**
+ * To rendering multiple versions of the API docs, this function is called for each version.
+ * @param apiVersionNum api version number, e.g. "2.0"
+ */
+async function renderApiDocumentation(apiVersionNum) {
+	const apiReportsDirectoryPath = path.resolve(__dirname, "..", "_api-extractor-temp", "_build");
```

Comment:
This needs to read from the directory for the corresponding version too, right? Otherwise we'll generate the same versions docs for each output directory, ignoring that version number we were given.

Reply:
Right. But it will fail the build with this change only... Should I include the version number change in this PR?

Reply:
Oh, I see. I think the way we fix that is by modifying `versions.json` before merging to only include 1.0 docs.

As it is, this will always generate both 1.0 and 2.0 docs, but both versions published to the website will actually be the 2.0 docs, which we don't want.

Reply:
Update this file will break current PR build. Marking down to update this place after https://github.com/microsoft/FluidFramework/pull/18677 merged.

Reply:
Merged from main. Updated.